### ISSUE TITLE
Unable to clone from Dashboard menu

### ISSUE THREAD

**Issue by RussKie** [gitextensions/gitextensions#5275]
**Do you want to request a *feature* or report a *bug*?**
bug / regression

**What is the current behavior?**
![image](https://user-images.githubusercontent.com/4403806/43563506-d0ce7fb4-9665-11e8-8c87-01ec1ad8fe89.png)

**What is the expected behavior?**
Able to clone and prompted to open, once the project is cloned

**Environment you encounter the issue:**

- GitExtensions version: 68ecad2ca3dde3db4a2644afcb0780894e666167

**Comment by RussKie** [comment]
The regression is introduced in 87f3f1199310639967b304109a19d44ae70b0bfb

```diff
--- a/GitUI/FormProcess.cs
+++ b/GitUI/FormProcess.cs
@@ -37,7 +37,7 @@ public FormProcess(ConsoleOutputControl outputControl, [CanBeNull] string proces
             Remote = "";
             ProcessInput = input;
             WorkingDirectory = workingDirectory;
-            Text = Text + " (" + WorkingDirectory + ")";
+            Text += $" ({PathUtil.GetDisplayPath(WorkingDirectory)})";
 
             ConsoleOutput.ProcessExited += delegate { OnExit(ConsoleOutput.ExitCode); };
             ConsoleOutput.DataReceived += DataReceivedCore;
```

![image](https://user-images.githubusercontent.com/4403806/43581176-66b1fa76-969b-11e8-96bb-76dd2cb992d5.png)

**Comment by drewnoakes** [comment]
The root cause of the bug was introduced earlier still. `Try` methods should never throw.  Fixed in #5284.
